resharper/resharper-unity/src/CSharp/Daemon/Stages/CallGraphStage/CallGraphAbstractStage.cs

```diff
@@ -79,14 +83,28 @@ public override void Execute(Action<DaemonStageResult> committer)
             committer(new DaemonStageResult(highlightingConsumer.Highlightings));
         }
 
+        private void CheckForInterrupt(IHighlightingConsumer consumer)
+        {
```

Comment:
SeldomInterruptChecker?

Reply:
CSharpDaemonProcessBase.IsProcessingFinished checks interruptFlag, seldom has it's own interrupt cookie, optimization?